.github/workflows/e2e.yaml

```diff
@@ -62,8 +62,6 @@ jobs:
       with:
         path: bin
         key: ${{ runner.os }}-${{ runner.arch }}-${{ hashFiles('Makefile') }}-${{ steps.setup-go.outputs.go-version }}
-    - name: Install chainsaw
```

Comment:
the way our makefile is written this is not needed, actually it confuses which version is used.